### DISCUSSION TITLE
忍者龟：施莱德的复仇

### DISCUSSION THREAD

**Discussion by LIUCG92** [Gamuxorg/bbs#296] [投递和请求游戏]
游戏名称：忍者龟：施莱德的复仇
游戏名称(英文)： Teenage Mutant Ninja Turtles: Shredder's Revenge
下载地址 :（如果有的话）：https://store.steampowered.com/app/1361510/_/?l=schinese

**Comment by hyh19962008** [comment]
刚出的游戏还请支持正版。
